lib/cocoapods/bazel/target.rb

```diff
@@ -50,10 +50,8 @@ def bazel_label(relative_to: nil)
         package_basename = File.basename(package)
         if package == relative_to
           ":#{label}"
-        elsif package_basename == label
-          "//#{package}"
```

Comment:
Keeping this isn't strictly necessary, but it's a cosmetic improvement to the generated code. Bazel has a shorthand where if the target name matches the package name (or the repo name, for the root package) that's the implied target. Eg `//my/path/to/discord_ios:discord_ios` can be equivalently written as `//my/path/to/discord_ios`. And similarly `@PINCache//:PINCache` can be written as `@PINCache`. Leaving in the special case (adapted to the external repo pattern) makes the generated code a bit nicer/less verbose.

Reply:
Yeah, I think it's generally less readable, but I'm fine doing that (because it's also go things where it creates targets for files in the package, and it's not always obvious whether it's a rule, or a file that something's using).